docs/catalogs/catalog-redis.md

```diff
@@ -10,22 +10,25 @@ title: Redis
 All Redis servers are 100% dedicated to your environment.
 
 1. Login to your KintoHub account
-2. Click **Create Service** at the top right of your environment
-3. Click the **From Catalog** tab at the top right
-4. Click the **Redis** option in the list
+2. Click on the **Create Service** at the top right of your environment
```

Comment:
action : Click on the **Create Service** button

Reply:
re: Done